recipes/eo-learn/LICENSE

```diff
@@ -0,0 +1,28 @@
+MIT License
```

Comment:
I just noticed that the license file is in the pypi tarball, so this file can be removed from here.

Reply:
One more minor update @mwilson8 , sorry about that :)